x-pack/legacy/plugins/transform/public/app/sections/create_transform/components/step_define/step_define_form.tsx

```diff
@@ -89,6 +92,58 @@ export function getDefaultStepDefineState(
     valid: false,
   };
 }
+
+export function applyTransformConfigToDefineState(
+  state: StepDefineExposedState,
+  transformConfig?: TransformPivotConfig
+): StepDefineExposedState {
+  // apply the transform configuration to wizard DEFINE state
+  if (transformConfig !== undefined) {
+    // transform aggregations config to wizard state
+    state.aggList = Object.keys(transformConfig.pivot.aggregations).reduce((aggList, aggName) => {
+      const aggConfig = transformConfig.pivot.aggregations[aggName] as Dictionary<any>;
+      const agg = Object.keys(aggConfig)[0];
+      aggList[aggName] = {
+        agg: agg as PIVOT_SUPPORTED_AGGS,
+        aggName,
+        dropDownName: aggName,
+        ...aggConfig[agg],
```

Comment:
nit: I think spread should always be on top, to be sure you override the values

Reply:
Fixed in c5ae56a.